src/containers/AccountConnection.jsx

```diff
@@ -120,15 +121,15 @@ class AccountConnection extends Component {
       })
   }
 
-  updateAccount (connector, account, values) {
+  updateAccount (connector, account, values, successTimeout) {
     Object.assign(account.auth, values)
 
     this.setState({ submitting: true })
 
     return this.store.updateAccount(connector, account, values)
     .then(account => {
       this.setState({ account: account })
-      return this.store.runAccount(connector, account)
+      return this.store.runAccount(connector, account, successTimeout)
```

Comment:
Is the lib `konnectors.run` correctly handling the case when `successTimeout` is `undefined`? As far as I can read here, I think it will break in the `waitForJobFinished` since we don't have default value anymore.